### ISSUE TITLE
humanize function not found

### ISSUE THREAD

**Issue by FiloSpaTeam** [efficiently/larasset#15]
When i use image_tag, i have this in console:
![err](https://cloud.githubusercontent.com/assets/2974561/8480761/dc3e1c5a-20df-11e5-9618-4b1bd00cf57f.png)

**Comment by tortuetorche** [comment]
Hi @FiloSpaTeam,

You're right, I'm going to add this helper.

Cheers,
Tortue Torche

**Comment by FiloSpaTeam** [comment]
;-) I found this function and added in my global.php

```
    function humanize($str) {
     
    $str = trim(strtolower($str));
    $str = preg_replace('/[^a-z0-9\s+]/', '', $str);
    $str = preg_replace('/\s+/', ' ', $str);
    $str = explode(' ', $str);
     
    $str = array_map('ucwords', $str);
     
    return implode(' ', $str);
    }
```

**Comment by tortuetorche** [comment]
Good catch, but it'll be easier for other users to have it bundled in Larasset.

**Comment by FiloSpaTeam** [comment]
yeah ;D when bundled, i'll remove my function! thanks!

**Comment by FiloSpaTeam** [comment]
Thanks! :dancer:

**Comment by tortuetorche** [comment]
You're welcome, I'm going to backport this fix for Larasset 0.9 and 1.0
